db/model/Gdoc/enrichedToMarkdown.ts

```diff
@@ -96,6 +102,15 @@ export function stripCustomMarkdownComponents(content: string): string {
         const regex = new RegExp(`<${componentName}[^\n]*?/>`, "g")
         strippedContent = strippedContent.replace(regex, "")
     }
+    for (const componentName of Object.values(
+        CUSTOM_MULTILINE_MARKDOWN_COMPONENTS
+    )) {
+        const regex = new RegExp(
+            `<${componentName}[\\s\\S]*?<\\/${componentName}>`,
```

Comment:
it seems a bit arbitrary that multi-line components support their opening tag being spread over multiple lines when the self closing ones specifically forbid it. Not an issue, but the inconsistency gave me pause.